docusearch/management/commands/import_documents.py

```diff
@@ -67,10 +73,10 @@ def create_document(document, release_slug):
 
     d.release_agency_slug = release_slug
 
+    doc_file = File(open(doc_path, 'rb'))
     filename = os.path.basename(doc_path)
-    doc_path = 'docusearch/pdfs/%s' % filename
-    d.path = doc_path
-    d.save()
+
+    d.original_file.save(filename, doc_file, save=True)
```

Comment:
I just double-checked to make sure the problem wasn't my key. It looks like I can update to `18f-foia-docs`. [test doc](https://s3.amazonaws.com/18f-foia-docs/README_test.md) 
```bash
(krang)GSA-xs-MacBook-Pro:foia-hub gabrielramirez$ python manage.py import_documents
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "/Users/gabrielramirez/Documents/FOIA/projectkrang/foia-hub/docusearch/management/commands/import_documents.py", line 142, in handle
    process_agency(documents_directory, agency)
  File "/Users/gabrielramirez/Documents/FOIA/projectkrang/foia-hub/docusearch/management/commands/import_documents.py", line 122, in process_agency
    process_office(agency_directory, agency, d)
  File "/Users/gabrielramirez/Documents/FOIA/projectkrang/foia-hub/docusearch/management/commands/import_documents.py", line 102, in process_office
    create_document(document, office_slug)
  File "/Users/gabrielramirez/Documents/FOIA/projectkrang/foia-hub/docusearch/management/commands/import_documents.py", line 89, in create_document
    d.original_file.save(filename, doc_file, save=True)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/django/db/models/fields/files.py", line 89, in save
    self.name = self.storage.save(name, content)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/django/core/files/storage.py", line 51, in save
    name = self._save(name, content)
  File "/Users/gabrielramirez/.virtualenvs/krang/src/django-storages/storages/backends/s3boto.py", line 275, in _save
    rewind=True, **kwargs)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/boto/s3/key.py", line 1293, in set_contents_from_file
    chunked_transfer=chunked_transfer, size=size)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/boto/s3/key.py", line 750, in send_file
    chunked_transfer=chunked_transfer, size=size)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/boto/s3/key.py", line 951, in _send_file_internal
    query_args=query_args
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/boto/s3/connection.py", line 664, in make_request
    retry_handler=retry_handler
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/boto/connection.py", line 1070, in make_request
    retry_handler=retry_handler)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/boto/connection.py", line 1029, in _mexe
    raise ex
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/boto/connection.py", line 939, in _mexe
    request.body, request.headers)
  File "/Users/gabrielramirez/.virtualenvs/krang/lib/python3.4/site-packages/boto/s3/key.py", line 844, in sender
    http_conn.send(chunk)
  File "/usr/local/Cellar/python3/3.4.2_1/Frameworks/Python.framework/Versions/3.4/lib/python3.4/http/client.py", line 890, in send
    self.sock.sendall(data)
  File "/usr/local/Cellar/python3/3.4.2_1/Frameworks/Python.framework/Versions/3.4/lib/python3.4/ssl.py", line 718, in sendall
    v = self.send(data[count:])
  File "/usr/local/Cellar/python3/3.4.2_1/Frameworks/Python.framework/Versions/3.4/lib/python3.4/ssl.py", line 679, in send
    v = self._sslobj.write(data)
BrokenPipeError: [Errno 32] Broken pipe
```